src/utils/filters.ts

```diff
@@ -37,11 +37,7 @@ export const filtersPanelSsrInit = async ({ query, ssrHelpers, req }: SSRProps)
             query: '',
             include: queryState.issuer,
         }),
-        ssrHelpers.project.suggestions.fetch({
-            take: filtersTakeCount,
```

Comment:
why did you remove limit?

Reply:
currently to prefetch selected filter we use `ssrHelpers.project.getByIds` endpoint (see next line)